.dockerignore

```diff
@@ -1,2 +1,3 @@
 data
-node_modules
\ No newline at end of file
+node_modules
+.git
```

Comment:
Please put new line at EOF for POSIX compatibility.

http://hiroakiuno.hatenablog.com/entry/20070409/p1
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
@lightnet328 ping

Reply:
Thanks. I agree with that and fixed it.